src/m365/search/commands/externalconnection/externalconnection-list.ts

```diff
@@ -18,12 +18,16 @@ class SearchExternalConnectionListCommand extends GraphItemsListCommand<any> {
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
     this
-      .getAllItems(`${this.resource}/v1.0/external/connections`, logger, true)
+      .getAllItems(`${this.resource}/${urlStub}`, logger, true)
```

Comment:
Where's `urlStub` defined?

Reply:
Hmmm, I must have done something wrong as it shouldn't have urlStub. This was the latest one that I committed - https://github.com/pnp/cli-microsoft365/blob/4b0f90bd59b403c4da285e09b8bfe92c136e2f63/src/m365/search/commands/externalconnection/externalconnection-list.ts.

Reply:
It seems to be okay now. Glitch in the matrix perhaps?